docs/source/profiler.rst

```diff
@@ -42,4 +42,4 @@ Intel Instrumentation and Tracing Technology APIs
 .. for tracking purposes
 .. py:module:: torch.profiler.itt
 .. py:module:: torch.profiler.profiler
-.. py:module:: torch.profiler.python_tracer
\ No newline at end of file
+.. py:module:: torch.profiler.python_tracer
```

Comment:
What was changed on that line?

Reply:
new line stuff there. removed accordlingly.